melvs/melvsboot/src/main/java/com/melvs/melvsboot/GetList.java

```diff
@@ -0,0 +1,41 @@
+package com.melvs.melvsboot;
+
+
+public class GetList {
+
+//string variable
+    private String id;
+    private String name;
+    private String description;
+
+//to initialize
+    public GetList(){
```

Comment:
This is called contructor. This is defined so you can call new GetList()

Reply:
thanks, I've forgotten how constructors looked like in Java. I understand now 🙏